vars/main.yml

```diff
@@ -17,6 +17,7 @@ __ha_cluster_fullstack_node_packages:
   - libknet1-plugins-all
   - pacemaker
   - pcs
+  - openssl  # used in the role for generating preshared keys
```

Comment:
You shouldn't pub `openssl` here unless `openssl` is also required on the *managed* nodes.  The usage in this PR is on the *control* node.

Reply:
When I drop `openssl` here, tests fail on my machines like this (I removed `no_log: true` so that the error message is visible):
```
TASK [linux-system-roles.ha_cluster : Generate fence_xvm.key using OpenSSL]
fatal: [rh84-node2]: FAILED! => {"changed": false, "cmd": "openssl rand -base64 512", "msg": "[Errno 2] No such file or directory: b'openssl': b'openssl'", "rc": 2, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```
`rh84-node2` is a managed node, controller node is `rh84-node1`. My intention was to use `openssl` on managed nodes in task `Generate {{ preshared_key_label }} using OpenSSL` in `tasks/presharedkey.yml`, this very pull request adds that. Maybe I am missing something here?

Reply:
> Maybe I am missing something here?

No, my bad. I thought you were only using openssl on the control node.